README.md

```diff
@@ -51,14 +51,24 @@ make plugins.json
 
 In addition to running as a standalone server, the Marketplace is also designed to run as a Lambda function, compiling the `plugins.json` database into the binary for immediate access without further configuration.
 
+### Automatic Deployment
+
+Changes merged to `master` are automatically deployed to https://api.staging.integrations.mattermost.com.
+
+Changes merged to `production` are automatically deployed to https://api.integrations.mattermost.com.
+
+When adding or updating the plugins database (or corresponding tooling), submit the changes directly to `production`, and then merge `production` immediately back to `master` to reduce unnecessary merge conflicts. All other changes should be committed directly to `master`. Changes pending release from `master` should be merged to `production` after qualification and in coordination with any supporting Mattermost server release.
+
+### Manual Deployment
+
 Simply run the following:
 
 ```
-$ make deploy-lambda
+$ SLS_STAGE=staging make deploy-lambda
```

Comment:
for manual production deployment that may need to happen for any reason, will be done through AWS?

Reply:
Apologies, I'm not sure I fully understand the question -- if the auto deploy fails, this can be done via serverless with the appropriate AWS credentials, yes.

Reply:
Sorry, yes was wondering if auto-deploy fails, how would we deploy to production in that case. Cool looks like we should be able to do that using AWS. 👍